### ISSUE TITLE
Please add NeurIPS 2019 paper

### ISSUE THREAD

**Issue by negrinho** [D-X-Y/Awesome-NAS#10]
Towards modular and programmable architecture search
Paper: https://arxiv.org/abs/1909.13404
Code: https://github.com/negrinho/deep_architect

Thank you

**Comment by D-X-Y** [comment]
Nice paper, added it. Would you mind to let me know the type of its searching algorithm?

**Comment by negrinho** [comment]
The paper is more focused on designing a language to encode search spaces than on search algorithms. The language gives you a well-defined API with which search algorithms can interact with search spaces. This means that the implementations of search algorithms and search spaces can be decoupled to some extent. The code link I shared contains the implementation of the language and the framework that we built around it. In conclusion, this work is not about a new search algorithm or search space, but about how the introduction of this language can enable modularity in architecture search implementations. Hopefully that clarifies it :).
